index.js

```diff
@@ -48,9 +49,15 @@ function factory(prng) {
     return encodeTime(seedTime, TIME_LEN) + encodeRandom(RANDOM_LEN)
   }
 
+  function lower(seedTime) {
+    ENCODING = ENCODING_LOWER
```

Comment:
This will change the value of `ENCODING` globally, thereby using the lowercase encoding regardless of which function is called.

To illustrate:

```javascript
ulid()
> A238AB...

ulid.lower()
> r810de...

ulid()
> e762jkm // wrong output
```